bot/src/abilities.ts

```diff
@@ -157,10 +157,26 @@ export const abilities = [
     }
   },
   {
+    name: 'END_OF_ORDER',
+    traces: [
+      {
+        slotName: 'END_OF_ORDER_TOKEN'
+      }
+    ],
+    onComplete: async (submittedData, { read }) => {
+      const convoState = await read()
+      const products = convoState.products || []
+
+      if (products.length === 0) {
+        return `You do not have any products in your order!`
+      }
+      return products.map((product: Product) => `${product.quantity} ${product.name}`).join(', ')
+    }
+  },  {
     name: 'None',
     traces: [
       {
-        slotName: 'END_OF_ORDER'
+        slotName: 'AFFIRMATIVE'
```

Comment:
What are examples of the NLU model returning `AFFIRMATIVE`? Trying to understand why this is mapping to "None" ability.

Reply:
There was a subtle difference between Michael's NLU and Arman's -- for single-word utterances last week, such as "done", Michael's would return intent:"None", entity:"END_OF_ORDER", and Arman's returned intent:"None", token:"AFFIRMATIVE".  Your question is valid though